lib/mcb/commands/sites/geocode.rb

```diff
@@ -19,13 +20,13 @@
     Site
       .where(id: args.to_a)
       .find_each(batch_size: opts[:batch_size]) do |site|
-        MCB.geocode(obj: site, sleep: opts[:sleep])
+        MCB.geocode(obj: site, sleep: opts[:sleep], force: opts[:force])
```

Comment:
Will the force option be applied to Providers or will this be in a separate PR?

Reply:
That would be a different PR. This trello card was only for sites.